projects/plugins/jetpack/extensions/blocks/tiled-gallery/layout/mosaic/index.js

```diff
@@ -84,14 +84,19 @@ export default class Mosaic extends Component {
 	}
 
 	render() {
-		const { align, columns, images, layoutStyle, renderedImages, columnWidths } = this.props;
+		const { align, columns, images, layoutStyle, renderedImages } = this.props;
 
 		const ratios = imagesToRatios( images );
 		const rows =
 			'columns' === layoutStyle
 				? ratiosToColumns( ratios, columns )
 				: ratiosToMosaicRows( ratios, { isWide: [ 'full', 'wide' ].includes( align ) } );
 
+		const columnWidths = Platform.select( {
+			web: () => this.props.columnWidths,
+			native: () => getColumnWidths( rows, renderedImages, 1000 ),
```

Comment:
This value of `1000` that's hard-coded here slipped through in [this PR](https://github.com/Automattic/jetpack/pull/21767/files#diff-81cc84c2318b32fb5e688103bf319a32d251ca08ebd147ad5decb649ce6761c9R97).

Reply:
@SiobhyB would you agree we can leave this in?

Reply:
@guarani, I agree the reasoning makes sense here 👍